src/ert/scheduler/lsf_driver.py

```diff
@@ -127,14 +127,55 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, JobState]:
 
 
 def build_resource_requirement_string(
-    exclude_hosts: Sequence[str], resource_requirement: Optional[str]
+    exclude_hosts: Sequence[str],
+    realization_memory: int,
+    resource_requirement: str = "",
```

Comment:
Should this be `Optional[str]` as we are providing a default value?

Reply:
Optional implies that we allow `None` as a value, which we don't here (if you supply `None`, it will crash).